internal/setup.py

```diff
@@ -0,0 +1,54 @@
+#! /usr/bin/env python
+# -*- coding: utf-8 -*-
+try:
+    from setuptools import setup
+except ImportError:
+    from distutils.core import setup
+import setuptools
+
+"""
+1. cd internal
+2. python setup.py sdist
+3. twine upload dist/*
+"""
+setup(
+    name='kclvm',
+    author='KCL Authors',
+    version='0.4.2.2',
+    license='Apache License 2.0',
+
+    description='KCLVM', 
+    long_description='''A constraint-based record & functional language mainly used in configuration and policy scenarios.''',
+    author_email='',  
+    url='https://kusionstack.io/',  
+    packages=setuptools.find_packages(),
```

Comment:
There is no dependency on some resource files added: such as `checksum.txt`.

Reply:
Done.  Packing files are controlled by the MANIFEST.in  file, now the package including the entire kclvm directory.